app/core/personutils.js

```diff
@@ -47,7 +47,7 @@ personUtils.isPatient = (person) => {
 
 personUtils.isClinicianAccount = (user) => {
   const userRoles = _.get(user, 'roles', []);
-  return _.intersection(userRoles, ['clinic', 'migrated_clinic', 'clinician']).length > 0 || user?.isClinicMember;
+  return _.intersection(userRoles, ['clinic', 'migrated_clinic', 'clinician']).length > 0 || user?.isClinicMember || false;
```

Comment:
```suggestion
  return _.intersection(userRoles, ['clinic', 'migrated_clinic', 'clinician']).length > 0 || !!user?.isClinicMember;
```

Could also force a boolean here on undefined `user` by using `!!`.  Your call.